formatters/githubformat/githubformat.go

```diff
@@ -60,15 +64,30 @@ func (f *Formatter) BeforeAll(i *manifest.Import) error {
 	}
 
 	for _, comment := range comments {
-		matches := fingerprintRegex.FindAllStringSubmatch(comment, -1)
+		matches := fingerprintRegex.FindAllStringSubmatch(comment.Body, -1)
 		for _, fingerprint := range matches {
-			f.existingComments[fingerprint[1]] = true
+			f.existingComments[fingerprint[1]] = comment
 		}
 	}
 	return nil
 }
 
 func (f *Formatter) AfterAll(i *manifest.Import) error {
+
+	//cleanup stale comments
```

Comment:
```suggestion
	// cleanup stale comments
```

super minor, just want consistent indentation on the comments.